README.md

```diff
@@ -219,8 +217,8 @@ Important note: don't try to insert values using
 
 ```python
 from airflow import DAG
-from airflow.sensors.clickhouse_sql_sensor import ClickHouseSqlSensor
-from airflow.operators.clickhouse_operator import ClickHouseOperator
+from airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor import ClickHouseSqlSensor
+from airflow_clickhouse_plugin.operators.clickhouse_operator import ClickHouseOperator
```

Comment:
Is it a recommended way to import plugins now? Why does former `from airflow.sensors.clickhouse_sql_sensor` not work anymore?

Reply:
It's a change on Airflow. All the external plugins have to be imported from the root of the module that you want to use.

Reply:
https://github.com/apache/airflow/blob/2.0.0/UPDATING.md#custom-executors-is-loaded-using-full-import-path here a link that states that change.

Reply:
Looks like you have shared not a correct link, so I will just duplicate it here for the record: https://github.com/apache/airflow/blob/2.0.0/UPDATING.md#adding-operators-and-sensors-via-plugins-is-no-longer-supported

Yes, sounds clear to me, all related to legacy plugin system can be now dropped.

Reply:
Could you please double check and confirm that you have changed it everywhere within the repo? Can be checked by searching for `airflow\.\w+\.clickhouse` pattern.